clusterous/helpers.py

```diff
@@ -116,7 +116,7 @@ class SSHTunnel(object):
     class TunnelException(Exception):
         pass
 
-    def __init__(self, host, username, key_file, remote_port, host_port=22):
+    def __init__(self, host, username, key_file, remote_port, host_port=22000):
```

Comment:
I would much rather the default host_port be 22 rather than 22000, which is specific to our current use.